libvast/vast/system/reader_command.hpp

```diff
@@ -116,10 +116,13 @@ reader_command(const command::invocation& invocation, caf::actor_system& sys) {
     auto in = detail::make_input_stream(*file, uds);
     if (!in)
       return caf::make_message(std::move(in.error()));
+    if (*file == "-")
+      VAST_INFO_ANON("reader-command spawns reader for stdin");
+    else
+      VAST_INFO_ANON("reader-command spawns reader for file", *file);
```

Comment:
Can you move this below the instantiation of `reader`? Then you can use the same logging as in the parent branch, i.e., `VAST_INFO(reader, ...)`.

Reply:
That kinda defeats the point of this PR, as that is where the log message was located at previously, causing it not to get printed until input arrived.

Reply:
Ah, I didn't think that the constructor already performs blocking I/O. This needs some refactoring in the future.